app/components/noplugins.js

```diff
@@ -0,0 +1,33 @@
+const React = require('react')
+const Style = require('./style')
+const {shell} = require('electron')
+
+
+const NoPlugins = () => {
+  let css = `
+    body {
+      background-color: #5CC7B2;
+      margin: 0;
+    }
+    h2, h3 {
+      margin: 0;
+      padding: 10px;
+      display: block;
+      color: #fff;
+      text-align: center;
+    }
+    a {
+      color: #EBEBEB;
+    }
+  `
+
+  return (
+    <div className='container'>
+      <h2 className='loader'>Zazu requires plugins to work!</h2>
+      <h3 className='loader'>Have a look at the <a href='#' onClick={ () => { shell.openExternal('http://zazuapp.org/plugins/') } }>Zazu Plugins</a> page.</h3>
```

Comment:
The `onClick` method is an anti-pattern. Every cycle react views this`h3` as a new element because the function is uniquely created each time `NoPlugins` is called. Can this either be defined outside of the `NoPlugins`? Or convert this to the non-functional form?